Dockerfile.supervisor_workers

```diff
@@ -0,0 +1,39 @@
+FROM python:3.8.12
+# If you are deploying this container yourself,
+# include your own label. Example below.
+#LABEL James Kunstle <jkunstle@redhat.com>
```

Comment:
Is this still valid?

Reply:
... potentially not, good catch. will think about it.